src/feedback/feedback.js

```diff
@@ -683,4 +677,5 @@ window.Feedback.XHR.prototype.send = function(data, callback) {
         error: function() { callback(false) }
     });
 };
-})(window, document);
+
+export { Feedback };
```

Comment:
@niklasvh As part of a gradual process, the version of `feedback.js` that is in use here is evolving; my goal is for it to become an up-to-date, modularized and minimal version of the original (while ideally maintaining all of the functionality).

It could be a little while before I'm ready to upstream those changes back to you would you potentially be interested in incorporating any of these in future if & when I do get time?